modules/c/nrt/source/DateTime.c

```diff
@@ -515,6 +583,8 @@ NRTPRIV(char *) _NRT_strptime(const char *buf, const char *fmt, struct tm *tm,
     const char *bp;
     size_t len = 0;
     int alt_format, i, split_year = 0;
+    NRT_BOOL isYearSet = 0;
+    NRT_BOOL isDayOfYearSet = 0;
```

Comment:
Use NRT_TRUE or NRT_FALSE instead of assigning 0 or 1.

Reply:
Updated